packages/core/src/ux-configuration.ts

```diff
@@ -1,8 +1,8 @@
 import { inject } from 'aurelia-dependency-injection';
 import { getLogger } from 'aurelia-logging';
 import { Loader } from 'aurelia-loader';
-import { PLATFORM } from 'aurelia-pal';
 import { GlobalStyleEngine } from './styles/global-style-engine';
+import './styles/normalize.css';
```

Comment:
it's a difficult situation. I think it's unreasonable to distribute this. Maybe lets remove this line 
```
    PLATFORM.moduleName('!css-loader!./styles/normalize.css');
```
and just have
```ts
    PLATFORM.moduleName('./styles/normalize.css');
```
That together with the check
```ts
        if (text && typeof (text) === 'string') {
```
should be enough.

Reply:
but this is effectively what that code does, just by calling loader api... what if a customer wants to extract css into a separate css bundle?

Reply:
Isn't it the same for the combo `PLATFORM.moduleName('...')` + `if(...)` vs `import '...'`? If it's extracted then the if block does nothing, and it doesn't require us any instrumentation for the build process. I think `import '..'` will fail current build process as well.

Reply:
why would it? `npm run build` runs just fine, `normalize.css` gets copied to the dist folder

Reply:
@bigopon @3cp @ben-girardet so, what do we do?

Reply:
Ill test it tonight or tmr and get back to this

Reply:
@MaximBalaganskiy it works ... so it will generate this 
![image](https://user-images.githubusercontent.com/9994529/82109017-697dab00-9776-11ea-80e8-537f5fb71d15.png)

I haven't used it this way for so long that I didn't even know it. So that's great, we just need to ask user to make sure they handle `import './style/normalize.css'` in their build.


Works fine with the CLI bundler too
![image](https://user-images.githubusercontent.com/9994529/82109333-887d3c80-9778-11ea-8080-b39d45b2dc6a.png)

So i guess this is good. Maybe we can do it for everywhere else and start bundling with Rollup for simpler distribution again.

Thanks @MaximBalaganskiy 

Do you have any comments on this @EisenbergEffect?

Reply:
btw, we may need to tweak the id of the style element injected, so that it's always valid (i'm not entirely sure if the `_au_css:@....` is a valid id